include/ensmallen_bits/moead/decomposition_policies/tchebycheff_decomposition.hpp

```diff
@@ -57,7 +57,7 @@ class Tchebycheff
                                     const VecType& idealPoint,
                                     const VecType& candidateFitness)
   {
-    return arma::max(weight % arma::abs(candidateFitness - idealPoint));
+    return max(weight % abs(candidateFitness - idealPoint));
```

Comment:
@rcurtin Suggest to be careful with removing the arma:: prefix.  I encountered issues where C++ compilers were getting confused between std::max() and arma::max(), due to poor implementations of the corresponding standard C++ library and the byzantine mess known as ADL: https://en.cppreference.com/w/cpp/language/adl.html

This change would essentially increase the risk of an ADL blowup happening all over again.